test/assignment-tests.js

```diff
@@ -0,0 +1,80 @@
+import test from 'tape';
+import compose from '../examples/compose';
+
+const assignmentProps = [
+  'methods',
+  'properties',
+  'staticProperties',
+  'deepStaticProperties',
+  'propertyDescriptors',
+  'staticPropertyDescriptors'
+];
+
+const build = (num) => {
+  const composable = {};
+
+  assignmentProps.forEach(prop => {
+    composable[prop] = {
+      [num]: num,
+      override: num
+    };
+  });
+
+  return { compose: composable };
```

Comment:
I realize that there is also POJO support, but most of the tests are passing stamps, not POJOS. Unless I am misunderstanding your definition of a POJO here.

I'm treating `{ compose: composable }` as a stamp. I consider `composable` the POJO. Is that incorrect?